src/client/components/localisation/darwin_robot/view_model.ts

```diff
@@ -26,4 +27,17 @@ export class RobotViewModel {
     robot.add(BodyViewModel.of(this.model).body)
     return robot
   }
+
+  @computed
+  public get ball(): Object3D {
+    const ball = new Object3D()
+
+    ball.position.x = this.model.ball.position.x
+    ball.position.y = this.model.ball.position.y
+    ball.position.z = this.model.ball.position.z
```

Comment:
🚗 🔫  I think you could move these positions directly into `BallViewModel` and just have `return BallViewModel.of(this.model.ball).ball` here